apps/website/lint-staged.config.js

```diff
@@ -1,13 +1,10 @@
-import mainConfig from "../../lint-staged.config.js";
-
 /** @param {string[]} filenames */
-const buildEslintCommand = (filenames) =>
-  `next lint --fix ${filenames.map((file) => `--file ${file}`).join(" ")}`;
+const buildBiomeCommand = (filenames) =>
+  `biome check --no-errors-on-unmatched --apply ${filenames.join(" ")}`;
 
 const config = {
-  ...mainConfig,
```

Comment:
Why can't we continue to inherit the main config here still, given we're using biome everywhere?